build-system/server/lazy-build.js

```diff
@@ -139,7 +139,7 @@ async function preBuildRuntimeFiles() {
  * Pre-builds default extensions and ones requested via command line flags.
  */
 async function preBuildExtensions() {
-  const extensions = getExtensionsToBuild();
+  const extensions = getExtensionsToBuild(/* preBuild */ true);
```

Comment:
Is this related to this change?

Reply:
This fixes a bug in the lazy builder that affects storybook. (Launching storybook would cause all ~150 extensions to be built, but now we quickly start up the `gulp` server and build extensions on-demand.)